DEVEL.md

```diff
@@ -137,17 +139,19 @@ $ git clone https://github.com/<GitHub Username>/community-operators
 
 Generate bundle and build bundle image:
 ```
-$ make bundle OPERATOR_VERSION=0.X.Y CHANNELS=alpha DEFAULT_CHANNEL=alpha
+$ make bundle CHANNELS=alpha DEFAULT_CHANNEL=alpha
```

Comment:
is there a precondition that this is run in a release branch?

Reply:
Umm, if we put 'set-version' as the begining of 'make bundle', then I do not think so.

Reply:
Ah, actually true.

Then,,,
1. would we put 'TAG' there..?
or
2. would we just keep OPERATOR_VERSION in Makefile, but as OPERATOR_VERSION ?= $TAG?

Reply:
I thought first option would be good enough, but if you have different opinion, let me know.!